distribution/ddf/src/main/filtered-resources/etc/org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl.config

```diff
@@ -1,13 +1,13 @@
-ignoredApplications=[
-    "hibernate-osgi",
-    "hibernate-validator-osgi-features",
-    "install-profiles",
-    "kernel",
-    "opendj-embedded",
-    "org.ops4j.pax.cdi",
-    "org.ops4j.pax.jdbc",
-    "org.ops4j.pax.web",
-    "spring",
-    "standard",
-    "karaf-decanter"
-]
\ No newline at end of file
+ignoredApplications = [ \
+  "hibernate-osgi", \
+  "hibernate-validator-osgi-features", \
+  "install-profiles", \
+  "kernel", \
+  "opendj-embedded", \
+  "org.ops4j.pax.cdi", \
+  "org.ops4j.pax.jdbc", \
+  "org.ops4j.pax.web", \
+  "spring", \
+  "standard", \
+  "karaf-decanter", \
```

Comment:
❓ Do we need the extra `,` after the last element?

Reply:
The extra `,` is optional.

Reply:
Should we remove it to avoid any confusion?

Reply:
I like that these default config files match exactly how felix formats them. I'll add a note in the documentation.